spec/rubocop/cop/solidus/reimbursement_hook_deprecated_spec.rb

```diff
@@ -30,4 +30,20 @@
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Please remove reimbursement_success_hooks and reimbursement_failed_hooks. Subscribe to reimbursement_reimbursed event instead.
     RUBY
   end
+
+  context 'when target solidus version is lower than the cop minimum solidus version' do
+    let(:config) { RuboCop::Config.new('AllCops' => { 'TargetSolidusVersion' => '2.10' }) }
+
+    it 'avoid to register an offense when using `#reimbursement_success_hooks.each`' do
+      expect_no_offenses(<<~RUBY)
+        reimbursement_success_hooks.each
+      RUBY
+    end
+
+    it 'avoid to register an offense when using `#reimbursement_failed_hooks.each`' do
+      expect_no_offenses(<<~RUBY)
+        reimbursement_failed_hooks.each
+      RUBY
+    end
+  end
```

Comment:
It looks to me that these tests are not related with the change.
I would suggest to remove them, `minimum_solidus_version 2.11` it's just a configuration and the behavior should be tested on [add_offense](https://github.com/nebulab/rubocop-solidus/blob/a34f822ad28d6b56ea962146de10905e40749d9c/lib/rubocop/cop/mixin/target_solidus_version.rb#L63-L67) method.

Reply:
Tests removed after merging #42